kfs-core/src/main/java/edu/arizona/kfs/fp/batch/service/impl/BankTransactionsLoadServiceImpl.java

```diff
@@ -0,0 +1,258 @@
+
+        for (String filePath : filesToValidate) {
+            LOG.info("validateBankTransactionFiles() Beginning validation of file: " + filePath);
+            try {
+                CsvBankTransactionsInputFileType inputFile = getBatchInputFileType();
+                inputFile.setFileToProcess(filePath);
+                inputFile.deleteDoneFile();
+                boolean isFileValid = inputFile.validate( );
+                if (!isFileValid) {
+                    LOG.error("Errors occurred for " + filePath + "File will not be processed. Please see error report:" + inputFile.getErrorFilePath());
+                    result = false;
+                } else {
+                    inputFile.createValidatedFiles();
+                    //record processed file to avoid duplicate file uploads
+                    recordFileUpload(inputFile.getBankFileInfo());
+                }
+
+            } catch (RuntimeException e) {
+                //we swallow the exception so we can continue processing files, the errors should have been reported in the error log
+                LOG.error("Caught exception trying to load bank file: " + filePath, e);
+                result = false;
+            }
+        }
+
+
+        LOG.info("End validateBankTransactionFiles() with result=" + result);
+        return result;
+    }
+
+
+    /**
+     * Consolidates all the data across multiple bank transaction files in one file, skipping the first row of
+     * each file (which should be the validated batchTotal for all the transactions in the file)
+     *
+     * If there are no files validated available for processing, returns false so next in documentCreationJob step won't execute.
+     *
+     * @return True if the file consolidation was successful, false otherwise.
+     */
+    public boolean consolidateBankTransactionFiles() {
+        LOG.info("Beginning consolidation of all available bank input transaction files for Document Creation Job.");
+        boolean result = true;
+
+        // create a list of the files to process
+        List<String> filesToProcess = getBatchInputFileService().listInputFileNamesWithDoneFile(getBankTransactionsValidatedFileType());
+
+        // end this step with a false if no files are found
+        if (filesToProcess == null || filesToProcess.isEmpty()) {
+            LOG.info("End consolidateBankTransactionFiles() with FALSE since no files were found!");
+            return false;
+        }
+
+        try {
+            getBankTransactionsValidatedFileType().consolidateFiles( filesToProcess );
+        }
+        catch (Exception e) {
+            LOG.error("ERROR in method consolidateBankTransactionFiles. DocumentCreationJob will not continue!",e);
+            // instead of rethrowing the exception, try to end in a civilized way returning false so batch won't continue...
+            return false;
+        }
+
+        LOG.info("End consolidateBankTransactionFiles() with result=" + result);
+        return result;
```

Comment:
Result will always be false, might be better readability if line 100 is deleted, and just return false here; clear and concise on what can happen that way.